### PR TITLE
Water species

### PR DESCRIPTION
**Pull request by YouCanBeBetterThanThat**
# POST FEEDBACK PLEASE

new water species, I intend it to be a hard mode race ala plasmamen but bit harder; posting as draft for initial feedback to concept. Also would like name suggestions

currently they:
- require a wetsuit and helmet or they turn into a simplemob blob that can't harm anyone. any damage done in this form is transferred back as clone damage. hardsuits will also work
- require CO2 to breathe in both forms
- their limbs depend on their blood volume, which is water, like slimepeople but they will automatically regrow it back if they meet the requirements
- blood volume will be heavily impacted by most forms of damage
- they gib on death in both forms so they can't be revived in any way. They also have BADDNA so no prescanning either. Any way to revive them is considered a bug
- they take more damage from all sources
- reagents will not process so they can't be healed outside of a sleeper or bruise packs
- They have a massive slowdown
- In blob form they can be scooped back into human form by having an outside source use a wetsuit on them, I might make it so you can crawl into one but this should take quite a while for the blob
- Since they have clone damage when they transform back into human form I made it so the human form has passive clone damage healing, only minor but enough to heal wounds over time. This is to massively disincentivize going into blob form. 
- Immune to viruses, radiation and wounds. They would be immune to viruses anyways due to always having internals on but it did not make sense to me to have a single-cellular creature have viruses or wounds. 
- Can't be dismembered traditionally, however they will actually in practice lose limbs more than the average crew member due to losing water volume and therefore losing their limbs.
- Has no genes, in practice almost always a downside since they can't have SA/hulk/whatever.
- In practice has no blood so can't be pod cloned.

I'm missing:
- Sprites for both forms (I could probably make a decent codersprite for human form)
- sounds
- Different gib sprites
- I want to be able to make it so they have no organs and process things innately as more of a single-cell creature instead of multicell
- Thinking about adding a very very slow passive healing, maybe dependent on blood volume
- Thinking about converting all damage into 2x the burn damage to simplify healing
- Naming conventions (I can think of something)
- Wound code for flesh only mobs
- Moving in blob form SHOULD make the tiles wet and cause health to drop, lose water volume too. I'll need to be able to transfer water volume from blob form to human form, should be easy

Ideas:
- Chance for laser-type projectiles to refract (keep damage, maybe only 75%, too good?) after causing damage
- Item that applies waterproofing to a uniform to allow it to be a wetsuit, use components for this. Maybe from xenobio or a research item, maybe even a 1 TC traitor item
- Transfer all brute damage (Not burn, water gets hot) to the suit, make it burst after so much damage. System for this already exists in terms of covering up body parts
- YOU SHOULD BE ABLE TO SCOOP UP BLOB FORM WITH A BEAKER. Sentient reagent DO NOT DRINK
- Maybe they shouldn't need a helmet, main body needs supporting but head is fine
- Potassium interactions?

Roadmap:
- [x] Initial species datum
- [x] Gib on death
- [x] Blob form and going in and out of it
- [ ] Remove all organs and replace with innate processing
- [ ] Sprites for both forms
- [ ] Gib sprites
- [ ] Split into different files
- [ ] Modularize
- [ ] Wound code for flesh-only mobs
- [ ] LORE
- [ ] wetsuit sprites. for. every. department.
- [ ] Outfits

# Wiki Documentation

species will need documenting

# Changelog

:cl:  
rscadd: Nanotrasen has discovered a new species in the delta sector, more information coming later
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
It is an interesting idea, but waterpeople sound even weaker than preternis, since they take more damage from or sources and are unrevivable.

**Comment by YouCanBeBetterThanThat** [comment]
> It is an interesting idea, but waterpeople sound even weaker than preternis, since they take more damage from or sources and are unrevivable.

that's the plan

**Comment by ghost** [comment]
> > It is an interesting idea, but waterpeople sound even weaker than preternis, since they take more damage from or sources and are unrevivable.
> 
> that's the plan

Give them then permanent quiet mode and restrict them from security/miners/command roles

**Comment by YouCanBeBetterThanThat** [comment]
> > > It is an interesting idea, but waterpeople sound even weaker than preternis, since they take more damage from or sources and are unrevivable.
> > 
> > 
> > that's the plan
> 
> Give them then permanent quiet mode and restrict them from security/miners/command roles

No quiet mode but I will absolutely restrict them from security to avoid loot pinata situations. Command roles will either be CMO or RD if any

**Comment by NCRLouTenant** [comment]
OwO

**Comment by 00ze-cyclone** [comment]
They feel way to much like slimepeople in term of flavor and they are essentially simplemobs mechanically which isn't that interesting for medical. All the job in medical got absolutely nothing interesting to do with that species in its current concept, no cloning or mutation for the geneticist, immunity to viruses so the viro can go cry, chemist can't make any intersting mixes for them, no organ to do surgeries on or to do weird interspecies transplantation, no wound to treat and no reviving so only thing medbay can do with them is apply suture and mesh or use the sleeper when they're hurt a little.

Also by making them so weak they can't be sec or command which limit even more the posssibilities for interesting things to be done with that species.

It would be more interesting both mechanically and flavorwise for them to be organs floating in liquid in the same way plasmamen are a skeleton coated in gas. That would allow for them to give medbay something to do with them, maybe they could leave a dried corpse when dying so they can still be revived but it would require a special procedure, make their suit prevent the use of mesh and suture so instead it incentivise using chemicals which way more interesting, we could tie some of their characteristics to organs to allow for interesting experiments. Making them use so of the mechanics of other species make those mechanics more in depth instead of creating some new one that are only used once. And organs floating in water feel way more alien than just another slime species.

In their current concept it is seem somewhat interesting to play as them but not really WITH them, yes what I propose would make them less weak but in my opinion it is better to give a different and rich experience than simply an hard experience.

**Comment by Spacemanspark** [comment]
@00ze-cyclone Since this is a flavor discussion rather than a balance discussion, I'll put in my two cents here.

I disagree quite a bit with this notion; I don't think medical necessarily _has_ to have an interaction with this race, and your solution feels more akin to humanizing them than making them truly unique.

There aren't any negative impacts to them having a lack of interaction with the medbay--which, personally, I think has quite a bit of crew interface, perhaps even too much. In all honestly I'd like to see races such as preternis require medical less, and have more of a focus from robotics, for example. Medical presently has to see every single race that the station has, in one way or another. 

I certainly do think the concept could use a bit of tweaking, but I wouldn't go so far as to say it needs to fall in line with every other race--dynamic interactions come most often from unique features.

**Comment by YouCanBeBetterThanThat** [comment]
> They feel way to much like slimepeople in term of flavor and they are essentially simplemobs mechanically which isn't that interesting for medical. All the job in medical got absolutely nothing interesting to do with that species in its current concept, no cloning or mutation for the geneticist, immunity to viruses so the viro can go cry, chemist can't make any intersting mixes for them, no organ to do surgeries on or to do weird interspecies transplantation, no wound to treat and no reviving so only thing medbay can do with them is apply suture and mesh or use the sleeper when they're hurt a little.
> 
> Also by making them so weak they can't be sec or command which limit even more the posssibilities for interesting things to be done with that species.
> 
> It would be more interesting both mechanically and flavorwise for them to be organs floating in liquid in the same way plasmamen are a skeleton coated in gas. That would allow for them to give medbay something to do with them, maybe they could leave a dried corpse when dying so they can still be revived but it would require a special procedure, make their suit prevent the use of mesh and suture so instead it incentivise using chemicals which way more interesting, we could tie some of their characteristics to organs to allow for interesting experiments. Making them use so of the mechanics of other species make those mechanics more in depth instead of creating some new one that are only used once. And organs floating in water feel way more alien than just another slime species.
> 
> In their current concept it is seem somewhat interesting to play as them but not really WITH them, yes what I propose would make them less weak but in my opinion it is better to give a different and rich experience than simply an hard experience.

I could get down with a water organ tank type thing. I'd be happy to keep chemicals enabled in that case, however I really would like to keep permadeath. I could however make it so that they have more chances to get to medbay if critically injured by disabling soft critical for example.
My intent was not to make them similar to slimepeople at all, hence the environmental suit and the fact that they won't have organs (slimepeople do)

**Comment by YouCanBeBetterThanThat** [comment]
> @00ze-cyclone Since this is a flavor discussion rather than a balance discussion, I'll put in my two cents here.
> 
> I disagree quite a bit with this notion; I don't think medical necessarily _has_ to have an interaction with this race, and your solution feels more akin to humanizing them than making them truly unique.
> 
> There aren't any negative impacts to them having a lack of interaction with the medbay--which, personally, I think has quite a bit of crew interface, perhaps even too much. In all honestly I'd like to see races such as preternis require medical less, and have more of a focus from robotics, for example. Medical presently has to see every single race that the station has, in one way or another.
> 
> I certainly do think the concept could use a bit of tweaking, but I wouldn't go so far as to say it needs to fall in line with every other race--dynamic interactions come most often from unique features.

Would love to hear what you think should be tweaked

**Comment by 00ze-cyclone** [comment]
@Spacemanspark 
Medical purpose is to heal and improve the bodies of the crew, if someone should have an interface with the crew and their anatomy it should be them.
I don't say they need to be reskinned humans, I'm all for their unusual relation with their blood where it affect them more than just taking oxygen damage when loosing it, I like that their envirosuit instead of just killing you when you don't have it you instead go into another form with it's own mechanics. It's just that they won't have unique interaction because they literally are unaffected by many mechanics.
For exemple preternis: preternis aren't unaffected by chems(exception being medicines), instead they interact with them differently, purging them after 30 ticks and having special effects with oil and welder fuel, they aren't unable to be infected by viruses, instead preternis take more time to be infected but give resistance to virus, that combined with their purging of chemicals make vaccines almost required to cure them which change their relation with both chemistry and virology.

Uniqueness won't create interesting interaction when that uniqueness come from not interacting with large parts of the game mechanics, I don't want them to have the same relation as other species with the mechanics, I want them to have different interaction instead of simply having their mehanics that are completely disconnected from everything else and therefore won't be able to interact in any way.

**Comment by YouCanBeBetterThanThat** [comment]
> @Spacemanspark Medical purpose is to heal and improve the bodies of the crew, if someone should have an interface with the crew and their anatomy it should be them. I don't say they need to be reskinned humans, I'm all for their unusual relation with their blood where it affect them more than just taking oxygen damage when loosing it, I like that their envirosuit instead of just killing you when you don't have it you instead go into another form with it's own mechanics. It's just that they won't have unique interaction because they literally are unaffected by many mechanics. For exemple preternis: preternis aren't unaffected by chems(exception being medicines), instead they interact with them differently, purging them after 30 ticks and having special effects with oil and welder fuel, they aren't unable to be infected by viruses, instead preternis take more time to be infected but give resistance to virus, that combined with their purging of chemicals make vaccines almost required to cure them which change their relation with both chemistry and virology.
> 
> Uniqueness won't create interesting interaction when that uniqueness come from not interacting with large parts of the game mechanics, I don't want them to have the same relation as other species with the mechanics, I want them to have different interaction instead of simply having their mehanics that are completely disconnected from everything else and therefore won't be able to interact in any way.

Any ideas for interesting generic interactions with chems?

**Comment by 00ze-cyclone** [comment]
Fluorosurfacant could make them more resistant or maybe less while in their system, it could also make them loose lot of blood and spill out of their suit because of the foam, effectively making them bleed and acting like a poison.
Some pyrotechnic chems that make you burn could instead  make them faster or stronger while giving them burn damage because they are boiling.
Frost oil could make them become more solid as they are frozen which could have a variety of effect, maybe they would bleed less because they are solid, they could take less damage or being easier to wound because ice can break while water can't.
Acid could do organ damage because they aren't contained in blood vessels like in other species when injected so they directly attack them.

Having them being mostly made of liquid can also allow for interesting way that they metabolize or get chemicals applied to them.
We could make them recieve more chems through smoke and foam so if they are outside their suit the effect of grenades would be multiplied, potentially causing ODs, allowing for healing/poison grenades to have less units of each chemicals to have the same lenght of effect which would open the door to different grenades mixes.
We could make them apply through touch chemicals in them if they are present  which could allow for quick application of medicines or toxins through punching or grabbing (chemist waterperson that heal you with a hug/punch to the face).
They could change color depending of the chems in them or scoop up automatically any reagents on the floor when not wearing theeir suit, which would allow them to clean very quickly but at the risk of getting viruses from infected blood or damage from the sulfuric acid blood of polys or the liquid electricity of ethereals.

**Comment by YouCanBeBetterThanThat** [comment]
> Fluorosurfacant could make them more resistant or maybe less while in their system, it could also make them loose lot of blood and spill out of their suit because of the foam, effectively making them bleed and acting like a poison. Some pyrotechnic chems that make you burn could instead make them faster or stronger while giving them burn damage because they are boiling. Frost oil could make them become more solid as they are frozen which could have a variety of effect, maybe they would bleed less because they are solid, they could take less damage or being easier to wound because ice can break while water can't. Acid could do organ damage because they aren't contained in blood vessels like in other species when injected so they directly attack them.
> 
> Having them being mostly made of liquid can also allow for interesting way that they metabolize or get chemicals applied to them. We could make them recieve more chems through smoke and foam so if they are outside their suit the effect of grenades would be multiplied, potentially causing ODs, allowing for healing/poison grenades to have less units of each chemicals to have the same lenght of effect which would open the door to different grenades mixes. We could make them apply through touch chemicals in them if they are present which could allow for quick application of medicines or toxins through punching or grabbing (chemist waterperson that heal you with a hug/punch to the face). They could change color depending of the chems in them or scoop up automatically any reagents on the floor when not wearing theeir suit, which would allow them to clean very quickly but at the risk of getting viruses from infected blood or damage from the sulfuric acid blood of polys or the liquid electricity of ethereals.

Yeah I wanted to make it so that they would change color depending on reagent amount and the color of the reagent, overall I meant mechanics that effect every chem instead of chemical specific interactions.

**Comment by bruhlookatthisdood** [comment]
what if they take off the wetsuit thing in the pool

**Comment by YouCanBeBetterThanThat** [comment]
> what if they take off the wetsuit thing in the pool

dissolve

**Comment by bruhlookatthisdood** [comment]
> > what if they take off the wetsuit thing in the pool
> 
> dissolve

step 1: kill water person
step 2: throw in pool
step 3: ???
step 4: greentext

**Comment by YouCanBeBetterThanThat** [comment]
> > > what if they take off the wetsuit thing in the pool
> > 
> > 
> > dissolve
> 
> step 1: kill water person step 2: throw in pool step 3: ??? step 4: greentext

they gib on death

**Comment by neeshacark** [comment]
maybe make it so they can move on hightraction tiles (the kind that cant be slippery) without losing water volume?

**Comment by YouCanBeBetterThanThat** [comment]
> maybe make it so they can move on hightraction tiles (the kind that cant be slippery) without losing water volume?

Alright

**Comment by ghost** [comment]
Me as a changeling turning people into waterpeople for an easy kill

**Comment by YouCanBeBetterThanThat** [comment]
> Me as a changeling turning people into waterpeople for an easy kill

NOTRANSSTING is there

**Comment by maxion12345** [comment]
How would wounds work with this?

**Comment by ghost** [comment]
> NOTRANSSTING is there

Me as a changeling not turning people into waterpeople for an easy kill

**Comment by YouCanBeBetterThanThat** [comment]
> How would wounds work with this?

They shouldn't have wounds ideally

**Comment by Spacemanspark** [comment]
Whoop, sorry, got distracted when I meant to answer this sooner.

Personally, I think there could be a bit more of a focus on the wetsuit. Brute and burn damages don't necessarily make a lot of sense against the entity itself, so perhaps have that convert to damage against the wetsuit. Cloning and toxins can still be quite deadly to the entity itself (which I suppose still gives medical some contact with them), and perhaps cold tempatures could cause them to slow down considerably more than a regular carbon, maybe even stop them from moving entirely at specific tempuratures.

As for the simple mob form, perhaps it should have a water volume variable, which has a much higher emphasis over general hit points. Moving too much or being exposed to a lot of heat (evaporation) causes them to lose water, which also makes the sprite smaller and smaller. Running out of water causes it to be perma dead, but water can be restored by either sitting in place for a while, or having someone insert fluids into it. Cloning and toxins could still bypass all this, however.

Only problem I immediately see with this are how new wetsuits are made, and/ or how they're repaired.

**Comment by YouCanBeBetterThanThat** [comment]
I'll be adding cargo packages for the wetsuits and such, and you'll be able to live indefinently in blob form so it won't be an issue like that but I like the idea of it damaging the suit

**Comment by Simplehorror** [comment]
Sprites are a placeholder for now.
Would anyone be interested in merging this? Want to know now before I keep working on it

**Comment by YouCanBeBetterThanThat** [comment]
@JamieD1 would like a code review so I can make some improvements before splitting up into files. also want to know if you have any interest in this so I don't keep working on it if you're gonna close, thanks!

**Comment by YouCanBeBetterThanThat** [comment]
Should I make it so you spawn with regular clothes but they have the wetsuit component or make it so you can order wetsuit applicators from cargo to modify normal clothes, but you spawn in an envirosuit roundstart?

**Comment by NCRLouTenant** [comment]
> Should I make it so you spawn with regular clothes but they have the wetsuit component or make it so you can order wetsuit applicators from cargo to modify normal clothes, but you spawn in an envirosuit roundstart?

Start in an envirosuit but can modify normal clothes later sounds like the better option to me, just from like standpoint of working towards a goal as the round goes on.

**Comment by YouCanBeBetterThanThat** [comment]
Bulk of the code done, just need sprites now

**Comment by YouCanBeBetterThanThat** [comment]
Ready for code review not sprites

**Comment by Anvilman6** [comment]
I have mixed feelings across the board on this one

It's an impressive initiative to take, but having another playable species means more tech debt when things involving species are changed. Plus, I have to question how many people will even PLAY this new species, seeing as it has extreme complexity (and mainly downsides, which as I understand is the intention).

Ultimately, this should be reviewed carefully by the headcoders before it gets any larger or more intricate.

File: code/modules/mob/living/carbon/human/species_types/water.dm [hunk 1]
```diff
@@ -0,0 +1,355 @@
...
+		qdel(P)
+		
+
+/datum/component/wetsuit_holder/Destroy()
+	STOP_PROCESSING(SSfastprocess, src)
+	. = ..()
+
+/obj/item/clothing/under/wetsuit
+	name = "bodily integrity wetsuit"
+	desc = "The suit that holds placeholder water races together. Without a suit compatible of contorting them, they would quickly fall apart."
```
Comment (Anvilman6) [inline code/modules/mob/living/carbon/human/species_types/water.dm]:
```suggestion
	desc = "The suit that holds the placeholder water races together. Without a suit capable of containing them, they would quickly fall apart."
```

File: code/modules/mob/living/carbon/human/species_types/water.dm [hunk 2]
```diff
@@ -0,0 +1,355 @@
...
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	can_adjust = FALSE
+
+/obj/item/clothing/under/wetsuit/Initialize()
+	AddComponent(/datum/component/wetsuit_holder)
+	. = ..()
+
+/mob/living/simple_animal/waterblob
+	name = "slobbery mess"
+	desc = "Something that can only be described as a bubbling viscous puddle extending tendrils to wherever it moves."
```
Comment (Anvilman6) [inline code/modules/mob/living/carbon/human/species_types/water.dm]:
```suggestion
	desc = "Something that can only be described as a bubbling viscous puddle, extending tendrils to wherever it moves."
```

File: code/modules/mob/living/carbon/human/species_types/water.dm [hunk 3]
```diff
@@ -0,0 +1,355 @@
...
+			T.MakeSlippery(TURF_WET_WATER, 40)
+			water_volume -= WATER_VOLUME_SLIP_MOVE_AMOUNT
+
+/mob/living/simple_animal/waterblob/get_status_tab_items()
+	. = ..()
+	. += "Water volume: [water_volume]"
+
+/obj/item/wetsuit_applicator
+	name = "wetsuit applicator"
+	desc = "Some waterproofing paint combined with a nanomaterial to prevent pores combined with a special shaping material allows any uniform to be used as a wetsuit."
```
Comment (Anvilman6) [inline code/modules/mob/living/carbon/human/species_types/water.dm]:
```suggestion
	desc = "Some extremely powerful waterproofing paint combined with nanomaterials. Allows any uniform to be used as a wetsuit for placeholder water race!"
```

File: yogstation/code/game/objects/items/tanks/tank_types.dm [hunk 4]
```diff
@@ -0,0 +1,13 @@
+/obj/item/tank/internals/co2
+	name = "CO2 internals tank"
+	desc = "A tank of CO2 gas designed specifically for use as internals. You don't want to use it unless you breathe CO2!"
```
Comment (Anvilman6) [inline yogstation/code/game/objects/items/tanks/tank_types.dm]:
```suggestion
	desc = "A tank of CO2 gas, designed specifically for breathing use by placeholder water race. Exercise caution before use!"
```

File: code/modules/mob/living/carbon/human/species_types/water.dm [hunk 5]
```diff
@@ -0,0 +1,355 @@
...
+	icon_state = "spraycan"
+
+/obj/item/wetsuit_applicator/afterattack(atom/A, mob/user, proximity)
+	. = ..()
+	if (!istype(A, /obj/item/clothing/under))
+		to_chat(user, span_notice("You can't use this on something that isn't a uniform!"))
+	AddComponent(A, /datum/component/wetsuit_holder)
+	to_chat(user, span_notice("You use the applicator on [A]."))
```
Comment (Anvilman6) [inline code/modules/mob/living/carbon/human/species_types/water.dm]:
```suggestion
	to_chat(user, span_notice("You apply waterproofing materials on the [A]."))
```

File: code/modules/mob/living/carbon/human/species_types/water.dm [hunk 6]
```diff
@@ -0,0 +1,355 @@
...
+	if (istype(H.w_uniform, /obj/item/clothing/under))
+		AddComponent(H.w_uniform, /datum/component/wetsuit_holder)
+
+	H.put_in_r_hand(new /obj/item/tank/internals/co2/full(H))
+	H.internal = H.get_item_for_held_index(2)
+	H.update_internals_hud_icon(1)
+
+/datum/species/water/spec_death(gibbed, mob/living/carbon/human/H)
+	. = ..()
+	H.visible_message(span_danger("[H] turns into a fine blue mist!"), span_userdanger("You evaporate!"))
```
Comment (Anvilman6) [inline code/modules/mob/living/carbon/human/species_types/water.dm]:
```suggestion
	H.visible_message(span_danger("[H] evaporates into a fine blue mist!"), span_userdanger("You evaporate!"))
```

File: code/modules/mob/living/carbon/human/species_types/water.dm [hunk 7]
```diff
@@ -0,0 +1,355 @@
...
+	name = "wet ears"
+	desc = "A pair of translucent ears. You think you've heard this one before."	
+
+/obj/item/organ/tongue/water
+	name = "dripping tongue"
+	desc = "A tongue made of water. It is dripping. You look at it and feel disgusted."
+
+/obj/item/organ/liver/water
+	name = "chemical distributor"
+	desc = "Looks like something that distributes chemicals around the body."
```
Comment (Anvilman6) [inline code/modules/mob/living/carbon/human/species_types/water.dm]:
```suggestion
	desc = "Looks like something that filters toxic chemicals from the body."
```

File: code/modules/mob/living/carbon/human/species_types/water.dm [hunk 8]
```diff
@@ -0,0 +1,355 @@
+#define WATER_VOLUME_MAXIMUM 560
+#define WATER_VOLUME_LIMB_LOSS (WATER_VOLUME_MAXIMUM / 5)
+#define WATER_VOLUME_LIMB_LOSS_THRESHOLD (WATER_VOLUME_MAXIMUM - WATER_VOLUME_LIMB_LOSS)
+#define WATER_VOLUME_SLIP_MOVE_AMOUNT 1
+
+#define WETSUIT_MAX_INTEGRITY 100
+/obj/item/organ/lungs/water
+	name = "vile sponge"
+	desc = "A spongy mass that looks like it is designed to carbonate water, what is it doing in a body?"
```
Comment (Anvilman6) [inline code/modules/mob/living/carbon/human/species_types/water.dm]:
```suggestion
	desc = "A spongy mass that looks like it is designed to carbonate water. What is it doing in a body?"
```

File: code/game/objects/items/tanks/tank_types.dm [hunk 9]
```diff
@@ -176,3 +176,17 @@
 
 /obj/item/tank/internals/emergency_oxygen/double/empty/populate_gas()
 	return
+
+/obj/item/tank/internals/co2
+	name = "CO2 internals tank"
+	desc = "A tank of CO2 gas designed specifically for use as internals. You don't want to use it unless you breathe CO2!"
```
Comment (Anvilman6) [inline code/game/objects/items/tanks/tank_types.dm]:
```suggestion
	desc = "An internals tank of CO2 gas, designed specifically for use by placeholder water race. Exercise caution before breathing with this tank!"
```

**Review by Anvilman6 — CHANGES_REQUESTED** [review]
May as well leave an initial review while I'm here

**Comment by Gboster-0** [comment]
it would be fun if they would instantly turn into a blob if they were to be in the pool, cool idea for a hardmode species.

**Comment by KillerOrcaCora** [comment]
stale

**Comment by Beeper-Skeeper** [comment]
imagine if they were ever added, too bad they weren't

**Pull request closed without merging**